tests/baselines/reference/tsxInvokeComponentType.errors.txt

```diff
@@ -13,7 +13,7 @@ tests/cases/compiler/tsxInvokeComponentType.tsx(6,14): error TS2322: Type '{}' i
                  ~~~~
 !!! error TS2322: Type '{}' is not assignable to type 'IntrinsicAttributes & { someKey: string; } & { children?: ReactNode; }'.
 !!! error TS2322:   Type '{}' is not assignable to type '{ someKey: string; }'.
-!!! error TS2322:     Property 'someKey' is missing in type '{}'.
+!!! error TS2322:     Property 'someKey' is missing in type {}' but present in type '{ someKey: string; }'.

```

Comment:
I really was expecting something like "Type is missing properties ..." to be the first elaboration, and avoiding `IntrinsicAttributes` altogether.